### PR TITLE
misc meta update

### PR DESCRIPTION
**Pull request by casperdcl**
- sync with https://github.com/iterative/cml.dev
- `sweedom.us10.list-manage.com` -> `dvc.us10.list-manage.com`
  - [x] should be fine according to https://github.com/iterative/cml.dev/pull/84#discussion_r692431785
- fix repo url
- add author

### CODE REVIEW

**Review by jorgeorpinel — APPROVED** [review]
LGTM but I'd double check the changes to gatsby-config.js and package.json with @rogermparent

File: gatsby-config.js [hunk 1]
```diff
@@ -270,6 +270,7 @@ module.exports = {
   plugins,
   siteMetadata: {
     description,
+    author: 'Iterative',
```
Comment (rogermparent) [inline gatsby-config.js:273]: I'm not sure if this is actually used anywhere, I at least don't see it in the SEO component's query, though there's a case for having a single place to define this for our plans to make the docs engine more reusable.

**Review by rogermparent — APPROVED** [review]
Looks fine by me!

File: gatsby-config.js (hunk 1 above)
Comment (casperdcl) [reply gatsby-config.js:273]: :shrug: 'twas just copied over from cml.dev

**Pull request merged**